### ISSUE TITLE
Win 7?

### ISSUE THREAD

**Issue by harryray33** [greenshot/greenshot#478]
Does this work with Win 7?

Thanks

**Comment by FF-Brown** [comment]
tl;dr Probably yes but it's not guaranteed.

From the latest release newsletter from the owner:

Supported platforms:
As always Greenshot comes "as-is" with no guarantee whatsoever, supplying free software doesn't make it possible to go beyond that.
Due to the amount of work involved in testing Greenshot on different platforms, and needing hardware to do so, we cannot support all of them. What does this mean?

Windows 11 - Greenshot should work without big issues. We expect to upgrade our primary development systems towards it some time after release, and have a better support soon.
Windows 10 "current" - We develop & test Greenshot on the latest available version of Windows 10, which means you can pretty much assume it works on there and if not we can work on it.
Windows 10 "previous" - We don't have the hardware to have all supported Windows versions side-by-side, we try to keep the compatibility high. If there are very Windows specific bugs, we can look at it. In general we add new Windows features with a check for the version, preventing issues with older versions.
Windows 10 insiders - If someone is on an insider, and wants to use Greenshot, there is no guarantee. But please do report issues, and make sure you mention the build you are on. As insiders are the next "current", we will try to look at this.
Windows 7 - this is no longer supported by Microsoft, and we will not buy extended support licenses for older Windows, we rather motivate people to upgrade to a normal supported version. It should still work on Windows 7, but we can't guarantee this and functionality will be degraded compared to that on Windows 10.
Windows 8(.1) - Greenshot 1.2 mostly worked on it, thus we assume 1.3 will do so too, but we can no longer support this.
Windows Server version XYZ- We do not have the hardware nor licenses to test on this, do so at your own risk. You can report issues, but we cannot support specific Windows Server issues, unless someone wants to pay for the time, hardware & licenses.
Linux via Wine or CrossOver - Although we made some small changes to Greenshot to get basics working, it's not supported and to be honest it's a lot of work to get it running, and the result is very disappointed. We are looking towards a more platform independend Greenshot, but this will take time and doesn't have our priority.
MacOS - We have a version of Greenshot for MacOS in the appstore [here](https://apps.apple.com/us/app/greenshot/id1103915944).

**Comment by Lakritzator** [comment]
Greenshot 1.2 was developed on Windows XP & 7, so when you stick to that version this will work.
BUT there are no guaranties, and we won't be able to help if it doesn't.
With version 1.3 it MIGHT work, but we will not be able to test this.

@InfiniteProductions this sounds like some programs are running elevated, as Greenshot is (hopefully) not, it will not get the key presses when this program is active.

I will close this issue, as I believe it's answered.

P.S.
I'm very curious about the reasons why people are still using Windows 7, especially if that system is connected to the internet.
Personally, it would creep me out, to use an OS which doesn't get OS security fixes...
https://www.cvedetails.com/product/17153/Microsoft-Windows-7.html?vendor_id=26
